### DISCUSSION TITLE
求一个王者荣耀国际版本 honor of kings分流规则

### DISCUSSION THREAD

**Discussion by jjyung168** [blackmatrix7/ios_rule_script#1335] [Q&A]
求一个王者荣耀国际版本 honor of kings分流规则

**Comment by beaxqq** [comment]
> 求一个王者荣耀国际版本 honor of kings分流规则

也想要

**Comment by squarelimber** [comment]
# honor of knights 王者荣耀
DOMAIN-SUFFIX,hok-dr.intlgame.com
DOMAIN-SUFFIX,hok.intlgame.com
DOMAIN-SUFFIX,intlgame.com
DOMAIN-SUFFIX,sgameglobal.com 
DOMAIN-SUFFIX,appsflyersdk.com
DOMAIN-KEYWORD,sgameglobal.com
IP-CIDR, 183.192.65.101/16, no-resolve
IP-CIDR, 43.132.55.55/16, no-resolve

作者我自己，亲测可用

**Reply by zqhaqha** [reply]
> # 骑士荣耀 王者荣耀
> 域名后缀，hok-dr.intlgame.com 域名后缀，hok.intlgame.com 域名后缀，intlgame.com 域名 后缀，sgameglobal.com 域名后缀，appsflyersdk.com 域名关键字，sgameglobal.com IP-CIDR，183.192.65.101/16，无法解析 IP-CIDR，43.132.55.55/16，无法解析
> 
> 作者我自己，亲测可用

大佬有更新吗，我刚把这些加入代理，进游戏还是提示所在区域不可用

**Reply by squarelimber** [reply]
我这里访问没问题，你确定有放进去吗，在规则里搜索下

**Reply by xzjun** [reply]
亲测可以

**Reply by githubpitt** [reply]
大佬，请问这个有订阅链接吗？

**Reply by squarelimber** [reply]
订阅链接是你自己购买的，这个只是规则，放到规则集模板里就行。油管很多视频都有讲

**Reply by jqyisbest** [reply]
大佬，今天更新后好像不行了，打开提示当前区域不支持

**Comment by githubpitt** [comment]
我本来问的是分流规则的订阅链接，不过没关系，我想明白怎么怎么做一个了，谢谢大佬。

**Reply by heudisi** [reply]
垃圾游戏要他干嘛

**Reply by heudisi** [reply]
还不如决胜巅峰

**Reply by heudisi** [reply]
还不如决胜巅峰

**Comment by imthnio** [comment]
DOMAIN-SUFFIX,honorofkings.com
DOMAIN-SUFFIX,sgameglobal.com
DOMAIN,download.2.687247097.hokglobalgcloud.iegcom.com
DOMAIN,g-client-prod.cloud-control.smoba.qq.com
DOMAIN,hok-dr.intlgame.com
DOMAIN,hok-lbs.intlgame.com
DOMAIN,hok.intlgame.com
DOMAIN,hok.singaporepaya.com
DOMAIN,li-sg.intlgame.com
